src/components/GameView/SpectatorListMenu.vue

```diff
@@ -29,10 +29,10 @@
       data-cy="spectate-list-menu"
     >
       <v-list-item-title v-if="spectatingUsers.length > 0">
-        Spectators
```

Comment:
It feels like this should be an interpolated value with the count. @itsalaidbacklife?

Reply:
This is just a title that displays over the spectator names,  the count is displayed on the icon to open the menu